### ISSUE TITLE
`neofs-cli container list` with `-g` flag behaves unexpectedly 

### ISSUE THREAD

**Issue by carpawell** [TrueCloudLab/frostfs-node#12]
It never returns any container cause of just generated wallet that of course does not maintain any containers. I'd expect that command iether to behave the same way `neofs-adm morph list-containers` does (that would be hard cause `neofs-adm` accepts a morph endpoint directly while `neofs-cli` works with `SN`) or does not allow passing `-g` at all.

**Comment by fyrchik** [comment]
You can still provide an explicit `--owner`.

**Comment by carpawell** [comment]
@fyrchik, yes, i can. But using `-g` flag is still useless (without `--owner`), it would always have an empty output. Also, i know that missing owner [returns](https://github.com/TrueCloudLab/frostfs-contract/blob/4077921794f364fca37c14b7743d8419435f9a4c/container/container_contract.go#L409-L410) all the containers while `cli` says "--owner string          Owner of containers (omit to use owner from private key)". Mb i'm too dev that i know contact details but that is what i had: i wanted to get all containers and the first thing to try was passing `-g` flag without `--owner` (or with `nil` owner in other words).

**Comment by fyrchik** [comment]
Let's add a verbose log/warning/error about using `-g` without `--owner`.
